huxley/core/models.py

```diff
@@ -266,53 +266,56 @@ def update_assignments(cls, new_assignments):
 
         def add(committee, country, school, rejected):
             additions.append(cls(
-                committee_id=committee,
-                country_id=country,
-                school_id=school,
+                committee_id=committee.id,
+                country_id=country.id,
+                school_id=school.id,
                 rejected=rejected,
             ))
 
         def remove(assignment_data):
             deletions.append(assignment_data['id'])
 
         for committee, country, school, rejected in new_assignments:
-            # If one or more fields in the assignment are invalid, the first 
-            # three values are passed in as strings instead of ints
-            is_valid = type(committee) is int
-
             key = (committee, country)
             if key in assigned:
                 # Make sure that the same committee/country pair is not being
                 # given to more than one school in the upload
-                committee = str(Committee.objects.get(pk=committee).name) + ' - DUPLICATE ASSIGNMENT IN CSV'
-                country = str(Country.objects.get(pk=country).name) + ' - DUPLICATE ASSIGNMENT IN CSV'
-                school = str(School.objects.get(pk=school).name)
+                committee = str(committee.name) + ' - DUPLICATE ASSIGNMENT IN CSV'
```

Comment:
So this output ends up looking something like this: 
```
('Trevor School', 'WHO - DUPLICATE ASSIGNMENT IN CSV', 'United States - DUPLICATE ASSIGNMENT IN CSV')
```
I think it would be more informative to just say: 
```
('Trevor School', 'WHO', 'United States') - DUPLICATE ASSIGNMENT IN CSV
```